stytch/b2b/rbac.go

```diff
@@ -0,0 +1,74 @@
+package b2b
+
+// !!!
+// WARNING: This file is autogenerated
+// Only modify code within MANUAL() sections
+// or your changes may be overwritten later!
+// !!!
+
+import (
+	"context"
+	"time"
+
+	"github.com/stytchauth/stytch-go/v11/stytch"
+	"github.com/stytchauth/stytch-go/v11/stytch/b2b/rbac"
+)
+
+type RBACClient struct {
+	C stytch.Client
+}
+
+func NewRBACClient(c stytch.Client) *RBACClient {
+	return &RBACClient{
+		C: c,
+	}
+}
+
+func (c *RBACClient) Policy(
+	ctx context.Context,
+	body *rbac.PolicyParams,
+) (*rbac.PolicyResponse, error) {
+	var retVal rbac.PolicyResponse
+	err := c.C.NewRequest(
+		ctx,
+		"GET",
+		"/v1/b2b/rbac/policy",
+		nil,
+		nil,
+		&retVal,
+	)
+	return &retVal, err
+}
+
+// MANUAL(PolicyCache)(TYPES)
+
+type PolicyCache struct {
+	rbacClient    *RBACClient
+	policy        *rbac.Policy
+	lastUpdatedAt time.Time
+}
+
+const refreshCadence = 300 * time.Second
+
+func NewPolicyCache(rbacClient *RBACClient) *PolicyCache {
+	return &PolicyCache{rbacClient: rbacClient}
+}
+
+func (pc *PolicyCache) shouldRefreshPolicy() bool {
+	return time.Since(pc.lastUpdatedAt) > refreshCadence
+}
+
+func (pc *PolicyCache) Get(ctx context.Context) (*rbac.Policy, error) {
+	if pc.policy == nil || pc.shouldRefreshPolicy() {
```

Comment:
Let's go all the way here

```suggestion
	return pc.policy == nil || time.Since(pc.lastUpdatedAt) > refreshCadence
}

func (pc *PolicyCache) Get(ctx context.Context) (*rbac.Policy, error) {
	if pc.shouldRefreshPolicy() {
```